### PR TITLE
cmd/tailscale,ipn: minor fixes to tailscale lock commands

### PR DESCRIPTION
**Pull request by twitchyliquid64**
* Fix broken add/remove key commands
 * Make lock status display whether the node is signed

### CODE REVIEW

**Comment by twitchyliquid64** [comment]
Looks like

```shell
[xxx@tslptp:~/nl-test]$ sudo ./tailscale --socket n2/sock lock
Network-lock is ENABLED.

Our node IS NOT trusted by network-lock, and action is required to establish connectivity.
Run the following command on a node with a network-lock key:
	tailscale lock sign nodekey:ac5e5d12d149e6a16a4ef55cc382494ccb0324fad70f23ffe637ee68f9a0320b

our public-key: nlpub:edc803e0309179af976e5e07b77e22715c7c5738f125f01472eb198323e4319b

Keys trusted to make changes to network-lock:
	nlpub:fd140eafa9be1504e1f7f4e4a00cd521cd64fb5077947d44536edec2fd8479e1	5	
	nlpub:edc803e0309179af976e5e07b77e22715c7c5738f125f01472eb198323e4319b	1	(us)
```

and

```shell
[xxx@tslptp:~/nl-test]$ sudo ./tailscale --socket n1/sock lock
Network-lock is ENABLED.

Our node is trusted by network-lock.

our public-key: nlpub:fd140eafa9be1504e1f7f4e4a00cd521cd64fb5077947d44536edec2fd8479e1

Keys trusted to make changes to network-lock:
	nlpub:fd140eafa9be1504e1f7f4e4a00cd521cd64fb5077947d44536edec2fd8479e1	5	(us)
	nlpub:edc803e0309179af976e5e07b77e22715c7c5738f125f01472eb198323e4319b	1	
```

**Comment by twitchyliquid64** [comment]
Friendly ping! :)

**Review by sailorfrag — APPROVED** [review]

File: cmd/tailscale/cli/network-lock.go [hunk 1]
```diff
@@ -82,11 +82,46 @@ func runNetworkLockStatus(ctx context.Context, args []string) error {
 	} else {
 		fmt.Println("Network-lock is NOT enabled.")
 	}
+	fmt.Println()
+
+	if st.Enabled && st.NodeKey != nil {
+		if st.NodeKeySigned {
+			fmt.Println("Our node is trusted by network-lock.")
```
[the remark this answers is not in the document]
Comment (twitchyliquid64) [reply cmd/tailscale/cli/network-lock.go:89]: Done.

File: ipn/ipnlocal/network-lock.go [hunk 2]
```diff
@@ -321,21 +321,46 @@ func (b *LocalBackend) NetworkLockStatus() *ipnstate.NetworkLockStatus {
...
+	var selfAuthorized bool
+	if b.netMap != nil {
+		selfAuthorized = b.tka.authority.NodeKeyAuthorized(b.netMap.SelfNode.Key, b.netMap.SelfNode.KeySignature) == nil
+	}
+
+	keys := b.tka.authority.Keys()
+	outKeys := make([]ipnstate.TKAKey, len(keys))
+	for i, k := range keys {
+		outKeys[i] = ipnstate.TKAKey{
+			Key:      key.NLPublicFromRawUnsafe(k.Public),
```
[the remark this answers is not in the document]
Comment (twitchyliquid64) [reply ipn/ipnlocal/network-lock.go]: Done - I went about this by having a more strongly typed conversion for `Key` types (via `ed25519.PublicKey`)

**Pull request merged**